js/equal-heights-blocks.js

```diff
@@ -28,23 +34,29 @@
           // before they are in place, then they get positioned
           // wrong on first load.
           setTimeout(() => {
+            removeExistingHeights();
             handleGetHeights();
           }, 250);
         })
       }
 
-      layoutsWithCallOutBox.forEach(layout => {
-        equaliseHeightsOfTheseBlocks(layoutsWithCallOutBox, '.call-out-box');
-      });
-
-      layoutsWithIaBlocks.forEach(layout => {
-        equaliseHeightsOfTheseBlocks(layoutsWithIaBlocks, '.ia-block');
-      });
+      function handleEqualise() {
+        layoutsWithCallOutBox.forEach(layout => {
+          equaliseHeightsOfTheseBlocks(layoutsWithCallOutBox, '.call-out-box');
+        });
+        layoutsWithIaBlocks.forEach(layout => {
+          equaliseHeightsOfTheseBlocks(layoutsWithIaBlocks, '.ia-block');
+        });
+        layoutsWithTeaserBlocks.forEach(layout => {
+          equaliseHeightsOfTheseBlocks(layoutsWithTeaserBlocks, '.featured-teaser');
+        });
+      }
 
-      layoutsWithTeaserBlocks.forEach(layout => {
-        equaliseHeightsOfTheseBlocks(layoutsWithTeaserBlocks, '.featured-teaser');
-      });
+      handleEqualise();
 
+      window.addEventListener('resize', function(event) {
```

Comment:
This looks good @cjstevens78, but I think we should add Drupal's `debounce` in here, so it's not firing on every pixel change for resize. Maybe a debounce with a value of 200 or 250 would be enough to keep it as performant as possible.

Reply:
Hi @markconroy ive made the change thanks chris